### ISSUE TITLE
Option for writeMedia to log warnings and continue instead of returning at the first error

### ISSUE THREAD

**Issue by gbogarinb** [bmaupin/go-epub#63]
Coming from [https://github.com/lapwat/papeer/issues/12](https://github.com/lapwat/papeer/issues/12)
Is it possible to just log a warning instead of returning an error?
In my particular case, there seems to be a broken img link that is halting the whole process, I really don't care if a single image is not downloaded so I would still like for the process to continue.

**Comment by bmaupin** [comment]
Yes, this library needs to handle errors better. We have an open issue to improve error handling (#49) but that's for returning an error instead of using `panic`.

This seems like a feature request to disable the validation altogether so an error isn't thrown. I like the idea to have that as an option.
